apps/client/src/features/app-settings/panel/project-panel/ProjectListItem.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react';
+import { IconButton, Menu, MenuButton, MenuItem, MenuList } from '@chakra-ui/react';
+import { IoEllipsisHorizontal } from '@react-icons/all-files/io5/IoEllipsisHorizontal';
+
+import { invalidateAllCaches, maybeAxiosError } from '../../../../common/api/apiUtils';
+import { deleteProject, duplicateProject, loadProject, renameProject } from '../../../../common/api/ontimeApi';
+
+import ProjectForm, { ProjectFormValues } from './ProjectForm';
+import { EditMode } from './ProjectList';
+
+import style from './ProjectPanel.module.scss';
+
+interface ProjectListItemProps {
+  current?: boolean;
+  filename: string;
+  createdAt: string;
+  updatedAt: string;
+  onToggleEditMode: (editMode: EditMode, filename: string | null) => void;
+  onSubmit: () => void;
+  onRefetch: () => Promise<void>;
+  editingFilename: string | null;
+  editingMode: EditMode | null;
+}
+
+export default function ProjectListItem({
+  current,
+  createdAt,
+  editingFilename,
+  editingMode,
+  filename,
+  onRefetch,
+  onSubmit,
+  onToggleEditMode,
+  updatedAt,
+}: ProjectListItemProps) {
+  const [submitError, setSubmitError] = useState<string | null>(null);
+
+  const handleSubmitRename = async (values: ProjectFormValues) => {
+    try {
+      setSubmitError(null);
+
+      if (!values.filename) {
+        setSubmitError('Filename cannot be blank');
+        return;
+      }
+      await renameProject(filename, values.filename);
+      await onRefetch();
+      onSubmit();
+    } catch (error) {
+      setSubmitError(maybeAxiosError(error));
+    }
+  };
+
+  const handleSubmitDuplicate = async (values: ProjectFormValues) => {
+    try {
+      setSubmitError(null);
+
+      if (!values.filename) {
+        setSubmitError('Filename cannot be blank');
+        return;
+      }
+      await duplicateProject(filename, values.filename);
+      await onRefetch();
+      onSubmit();
+    } catch (error) {
+      setSubmitError(maybeAxiosError(error));
+    }
+  };
+
+  const handleToggleEditMode = (editMode: EditMode, filename: string | null) => {
+    setSubmitError(null);
+    onToggleEditMode(editMode, filename);
+  };
+
+  const handleCancel = () => {
+    handleToggleEditMode(null, null);
+  };
+
+  const isCurrentlyBeingEdited = editingMode && filename === editingFilename;
+
+  return (
+    <tr key={filename} className={current ? style.current : undefined}>
+      {isCurrentlyBeingEdited ? (
+        <td colSpan={99}>
+          <ProjectForm
+            action={editingMode}
+            filename={filename}
+            onSubmit={editingMode === 'duplicate' ? handleSubmitDuplicate : handleSubmitRename}
+            onCancel={handleCancel}
+            submitError={submitError}
+          />
+        </td>
+      ) : (
+        <>
+          <td>{filename}</td>
+          <td>{createdAt}</td>
+          <td>{updatedAt}</td>
```

Comment:
question: I was sure this used to be formatted? We dont need to do much fancy here, we can either use luxon or a simple `toLocaleDateString`

Reply:
should it only display the date then? or datetime?

Reply:
I think just creating the locale date like `23.08.19 16:12:30` is OK